main/ZgatewayBT.ino

```diff
@@ -585,14 +604,25 @@ void setupBT() {
   semaphoreCreateOrUpdateDevice = xSemaphoreCreateBinary();
   xSemaphoreGive(semaphoreCreateOrUpdateDevice);
 
+  BLEQueue = xQueueCreate(32, sizeof(NimBLEAdvertisedDevice*));
+
   BLEDevice::setScanDuplicateCacheSize(BLEScanDuplicateCacheSize);
   BLEDevice::init("");
 
+  xTaskCreatePinnedToCore(
+      procBLETask, /* Function to implement the task */
+      "procBLETask", /* Name of the task */
+      4096, /* Stack size in bytes */
```

Comment:
We'll need to find a sufficient size for this. Setting log level to trace will print the high water mark for this stack.